mouse-look.js

```diff
@@ -14,6 +14,8 @@ WL.registerComponent('mouse-look', {
     /** If "moveOnClick" is enabled, mouse button which should
      * be held down to control view */
     mouseButtonIndex: {type: WL.Type.Int, default: 0},
+    /** Enables pointer lock on "mouseDown" */
```

Comment:
For referencing the event "mousedown", it would be a lowercase "d" :)
Maybe to clarify, it could be `... "mousedown" even on WL.canvas.` ?
In theory we could have the HTML documentation automatically link that, but I don't think that works yet for the component docs 🤔